src/scss/foundations/hiding.scss

```diff
@@ -0,0 +1,63 @@
+////
+/// Hiding Rules define all rules for hiding content based on device width.
+/// @group HIDING
+////
+
+@use "../lib";
+@use "breakpoints";
+
+/// Hide an element from `$fromBreakpoint` to `$untilBreakpoint` value.
+/// Outside the indicated breakpoints the element will have the assigned display value.
+///
+/// @param {String} $fromBreakpoint - Name of start breakpoint
+/// @param {String} $untilBreakpoint - Name of end breakpoint
+/// @throw {Error}
+///
+/// @example scss - Hide Mixin
+///  // Hide the element for every breakpoints
+///  .element {
+///     @include hide();
+///  }
+///
+///  // Hide element from breakpoint medium
+///  .element {
+///     display: grid;
+///     @include hide(medium);
+///  }
+///
+///  // Hide element between breakpoint small and large
+///  .element {
+///     display: grid;
+///     @include hide(small, large);
+///  }
+/// @access public
+@mixin hide($fromBreakpoint: base, $untilBreakpoint: null) {
+  $_untilBreakpoint: if(lib.isNotNull($untilBreakpoint), $untilBreakpoint, xlarge);
+
+  @if(breakpoints.breakpoint($_untilBreakpoint) + 1 > breakpoints.breakpoint($fromBreakpoint)) {
```

Comment:
Visto che è la seconda volta che facciamo questo controllo in due punti diversi direi che possiamo creare una funzione in `breakpoints.scss` di questo tipo: 
```scss
@function isLowerThan($breakpoint, $upperBound) {
...
}

@function isGreaterThan($breakpoint, $lowerBound) {
...
}

@function isBetween($breakpoint, $lowerBound, $upperBound) {
 @return isLowerThan($breakpoint, $upperBound) and isGreaterThan($breakpoint, $upperBound);
}

```

Reply:
Le funzioni sono state create e controllano se il primo valore è maggiore o uguale, minore o uguale rispetto al secondo nelle prime due funzioni.

Reply:
Rifattorizzate in modo che controllino solo strettamente maggiore o strettamente minore